lldb/include/lldb/Target/Process.h

```diff
@@ -3169,6 +3176,7 @@ void PruneThreadPlans();
                            // m_currently_handling_do_on_removals are true,
                            // Resume will only request a resume, using this
                            // flag to check.
+  lldb::RunDirection m_last_run_direction;
```

Comment:
This should have a doc string.  I've been trying not to make classes that default initialize some ivars in the class declaration and some in the constructors.  That just seems super confusing to me.  So I personally wouldn't initialize this one here, or I'd go fix all the ones we always initialize the same way in the constructors.  But that's a matter of taste.

Reply:
> This should have a doc string. 

OK, will do.

> So I personally wouldn't initialize this one here,

Maybe I'm misunderstanding you, but I'm not initializing `m_last_run_direction` here.

Reply:
Ah I see. @clayborg suggested I initialize `m_last_run_direction` inline but fortunately I never got around to doing that :-).